ui/web-v2/apps/admin/src/components/LanguageSelect/LanguageSelect.tsx

```diff
@@ -0,0 +1,95 @@
+import { getSelectedLanguage, LanguageTypes } from '@/lang/getSelectedLanguage';
+import { Listbox, Transition } from '@headlessui/react';
+import { Translate, ArrowDropDown } from '@material-ui/icons';
+import React, { FC, Fragment, memo, useState } from 'react';
+
+import { classNames } from '../../utils/css';
+
+interface ILanguageItem {
+  readonly label: string;
+  readonly value: LanguageTypes;
+}
+
+const languageList: ILanguageItem[] = [
+  { label: '日本国', value: LanguageTypes.JAPAN },
+  { label: 'English', value: LanguageTypes.ENGLISH },
+];
+
+export const LanguageSelect: FC = memo(() => {
```

Comment:
To keep the consistency with the other files, can you rename the file to `index.tsx`?
And import it like this.

```ts
import { LanguageSelect } from '@/components/LanguageSelect';
```